recipes/wasmtime/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest)
+
+set(C_STANDARD 11)
```

Comment:
This is unused. Unless you meant `CMAKE_C_STANDARD`.

Reply:
Of yes, I missed perfix `CMAKE_`
I'll fix it

Reply:
Fixed